src/components/organisms/ChatSidebar/ChatSidebar.tsx

```diff
@@ -133,6 +139,17 @@ export const _ChatSidebar: React.FC<ChatSidebarProps> = ({ venue }) => {
           </button>
         )}
 
+        {!isShownHelpCenter && !isExpanded && (
+          <button
+            aria-label={"Create space"}
+            className="chat-sidebar__controller chat-sidebar__create-icon"
+            onClick={goToAdmin}
+          >
+            <FontAwesomeIcon icon={faChevronLeft} size="sm" />
+            <FontAwesomeIcon icon={faPen} size="lg" />
+          </button>
```

Comment:
[OOS]

We should probably extract one component for all 4 of the existing sidebar buttons

Reply:
Doing that in https://github.com/sparkletown/sparkle/pull/2165